### ISSUE TITLE
cli tools don't work in linkml v1.3.9 docker image

### ISSUE THREAD

**Issue by capsulecorplab** [linkml/linkml#1103]
**Describe the bug**
For example, I encounter the following warning & error while attempting to print help text for `linkml-lint` using the [linkml v1.3.9 docker image](https://hub.docker.com/layers/linkml/linkml/v1.3.9/images/sha256-8bd3916f715d18d2cd0fc7face15cbddb6bfc1354f31fb9c38548a3ed4a29351?context=explore)
 
```
$ docker run -v $PWD:/work -w /work -ti linkml/linkml:v1.3.9 linkml-lint --help
WARNING: The requested image's platform (linux/arm64/v8) does not match the detected host platform (linux/amd64) and no specific platform was requested
exec /usr/local/bin/linkml-lint: exec format error
```

**To Reproduce**
1. run `docker run -v $PWD:/work -w /work -ti linkml/linkml:v1.3.9 linkml-lint --help`
2. See error

**Expected behavior**
Should print help text for `linkml-lint`

**Additional context**
Discovered after attempting to rerun CI following #1101. See https://github.com/capsulecorplab/m30ml/actions/runs/3446463568/jobs/5751396091#step:3:72

**Comment by capsulecorplab** [comment]
I did a local build of the linkml docker image from [v1.3.13](https://github.com/linkml/linkml/tree/v1.3.13) and the issue seems to have been resolved. @cmungall may I request that a [v1.3.13](https://github.com/linkml/linkml/tree/v1.3.13) version of the linkml docker image be published to dockerhub? I can't seem to run any service services via the v1.3.9 image, otherwise

**Comment by capsulecorplab** [comment]
OBE as per linkml docker image using tag `1.3.14` published from GitHub workflow #1120
